conda_forge_tick/migrators/cross_compile.py

```diff
@@ -114,6 +125,53 @@ def migrate(self, recipe_dir: str, attrs: "AttrsTypedDict", **kwargs: Any) -> No
                 f.write("".join(lines))
 
 
+class CrossPythonMigrator(CrossCompilationMigratorBase):
+    def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
+        host_reqs = attrs.get("requirements", {}).get("host", set())
+        build_reqs = attrs.get("requirements", {}).get("build", set())
+        return "python" not in host_reqs or "python" in build_reqs
```

Comment:
we usually include a call to super in here too, since that handles if the PR has already been issued

Reply:
Since this is a mini-migrator, is that relevant?

Reply:
Sorry I didn't realize this was a mini migrator